package.json

```diff
@@ -59,6 +62,7 @@
     "eslint-plugin-ember": "^6.3.0",
     "eslint-plugin-prettier": "3.0.1",
     "loader.js": "^4.7.0",
+    "marked": "^0.7.0",
```

Comment:
Some of the other realworld projects were using `marked`.

Compared to `markdown-it` (34.3KB gzipped) and `showdown` (26.1KB gzipped), `marked` was the lightest weight, coming in at 8.8KB gzipped.